tracing/tck/src/main/java/org/eclipse/microprofile/telemetry/tracing/tck/async/MpRestClientAsyncTest.java

```diff
@@ -96,6 +97,12 @@ public void testIntegrationWithMpRestClient() throws Exception {
         readSpans();
     }
 
+    @Test
+    public void testIntegrationWithMpRestClientAsyncError() throws Exception {
+        basicClient.get("/MpRestClientAsyncTestEndpoint/mpclientasyncerror");
+        readSpansError();
+    }
+
```

Comment:
There are no parameters on any of the tests in this class. Were they just forgotten or is there a reason we don't want to add them here?